packages/react-components/react-progress/stories/ProgressBar/index.stories.tsx

```diff
@@ -11,7 +11,7 @@ export { Shape } from './ProgressBarShape.stories';
 export { Thickness } from './ProgressBarThickness.stories';
 
 export default {
-  title: 'Preview Components/ProgressBar',
+  title: 'Components/ProgressBar',
```

Comment:
You should wrap your best practices with `<summary>` to make it an accordion like this:
```
<details>
<summary>
Best Practices
</summary>
// Your Do's and Dont's
</details>
```